content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+We have automatically migrated existing awesome-bevy entries, but we encourage creators to customize them! If you are working on something Bevy related, you are highly encouraged to [add a Bevy Assets entry](https://github.com/bevyengine/bevy-assets).
+
+## Dual MIT / Apache-2.0 License
+
+<div class="release-feature-authors">authors: @cart, @DJMcNab</div>
+
+Thanks to the relevant contributors ([all 246 of them](https://github.com/bevyengine/bevy/issues/2373)), Bevy is now dual licensed under MIT _and_ Apache-2.0, at the developers' option. This means developers have the flexibility to choose the license that best suits their specific needs. I want to stress that this is now _less_ restrictive than MIT-only, not more.
+
+I originally chose to license Bevy exclusively under MIT for a variety of reasons:
+
+* People and companies generally know and trust the MIT license more than any other license. Apache 2.0 is less known and trusted.
+* It is short and easy to understand
+* Many people aren't familiar with the "multiple license options ... choose your favorite" approach. I didn't want to scare people away unnecessarily.
+* Other open source engines like Godot have had a lot of success with MIT-only licensing
+
+However there were a variety of issues that have come up that make dual-licensing Bevy under both MIT and Apache-2.0 compelling:
+
+* The MIT license (arguably) requires binaries to reproduce countless copies of the same license boilerplate for every MIT library in use. Apache-2.0 allows us to compress the boilerplate into a single instance of the license.
+* The Apache-2.0 license has protections from patent trolls and an explicit contribution licensing clause.
+* The Rust ecosystem is largely Apache-2.0. Being available under that license is good for interoperation and opens the doors to upstreaming Bevy code into other projects (Rust, the async ecosystem, etc).
+* The Apache license is incompatible with GPLv2, but MIT is compatible. 
+
+## Bevy Org Changes
+
+### More pull request mergers!
+
+I've been at my scalability limits for awhile. It has been  * cough * ... challenging  ... to build the engine features I need to, review every single pull request quickly, and preserve my mental health. I've made it this far ... sometimes by overworking myself and sometimes by letting PRs sit unmerged for longer than I'd like. By scaling out, we can have our cake and eat it too!
+
+* @mockersf now has merge rights for "uncontroversial changes"
+* @alice-i-cecile now has merge rights for "uncontroversial documentation changes"
+
+### New issue labels
+
+After [much discussion about naming conventions and colors](https://github.com/bevyengine/bevy/issues/2256), we finally have a [fresh new set of issue labels](https://github.com/bevyengine/bevy/labels) (loosely inspired by the rust repo). The [Bevy Triage Team](https://github.com/orgs/bevyengine/teams/triage-team/members) can finally express themselves fully!
```

Comment:
Again, I'm on mobile so can't easily check, but is this team publicly viewable?

Reply:
It isn't, I get prompted to log in or get the error message "You’re not a member of any teams in this organization" if I log in with an account not in the bevy org.

Reply:
Yeah, we have the same issue in the CONTRIBUTING.md file :(

Reply:
Hmm yeah it looks like teams are fully private. I cant find a useful public link anywhere, so I'll just link to the relevant section of CONTRIBUTING.md